test-utils-bluetooth/src/androidLibMain/kotlin/MockDeviceWrapper.kt

```diff
@@ -35,7 +34,7 @@ class MockDeviceWrapper(
     setupMocks: Boolean = true
 ) : DeviceWrapper {
 
-    val gattWrappers = sharedMutableListOf<MockBluetoothGattWrapper>()
+    val gattWrappers = mutableListOf<MockBluetoothGattWrapper>()
```

Comment:
technically also better off thread safe, maybe we can just scan for all mutableList once we have a safe variant.